TeaDiary.business/Interfaces/ITeaRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using TeaDiary.domain.Models;
+
+namespace TeaDiary.business.Interfaces
+{
+    public interface ITeaRepository
+    {
+        IEnumerable<Tea> GetAll(int userId);
+        Tea GetById(int teaId);
```

Comment:
This doesn't have user restrictions. Maybe it should?

Reply:
Implemented